technical-reference/reference-technical-articles/technical-doc-xvs-bridge.md

```diff
@@ -0,0 +1,221 @@
+
+### 6.3. Limit the Amount of XVS Transfers <a name="limit-the-amount-of-XVS-transfers"></a>
+
+- Example: Limit the maximum XVS transfer to USD 1,000 in one transaction and USD 100,000 in one day. These limits can be adjusted using VIPs.
+
+### 6.4. Transfer Delays <a name="transfer-Delays"></a>
+
+- Configurable delay after XVS transfers to the target network by specifying a minimum number of blocks in the layer zero endpoint configuration.
+
+### 6.5. Token Controller Contract <a name="token-controller-contract"></a>
+
+- Token Controller contract within the XVS token deployed on the target network to blacklist addresses, preventing them from transferring or receiving XVS. Integrated with the ACM.
+
+### 6.6. Cap on Token Minting <a name="cap-on-token-minting"></a>
+
+- Cap on the amount of tokens that can be minted in the destination target network. This feature can be integrated in Token Controller.
+
+### Bridge Model
+
+The XVSProxyOFTDest contract serves as the bridge model. It will be authorized to mint and burn tokens in the destination chain. Limits on these actions will be set by Governance or Guardian.
+
+## 7. Contract Details <a name="contract-details"></a>
+
+Here, we provide more details about the key contracts used in the Bridge:
+
+### 7.1. XVSBridgeAdmin <a name="xvsbridgeadmin"></a>
+
+- XVSBridgeAdmin is the admin contract for the bridge, ensuring proper setup.
+- It contains a `functionRegistry` mapping for function signatures, allowing the contract to call corresponding methods in destination contracts after ensuring access control permissions.
+- Ownership transfers for XVSBridgeAdmin and Bridge can be executed via the `transferOwnership` and `transferBridgeOwnership` methods respectively.
+
+### 7.2. XVSProxySrc <a name="xvsproxysrc"></a>
+
+- XVSProxySrc extends the BaseOFTV2 contract and includes custom logic for token transfers.
+- It overrides the `_debitFrom` and `_creditTo` functions, checking transaction limits and user eligibility.
+- It enforces transaction limits, tracks 24-hour window limits, and allows whitelisting of users.
+- XVSProxySrc can be paused and resumed in emergencies.
+
+### 7.3. XVSProxyDest <a name="xvsproxydest"></a>
+
+- XVSProxyDest is similar to XVSProxySrc but with specific differences.
+- Transaction limits are enforced primarily for outbound amounts only in the source chain.
+- It overrides the `debitFrom` function to include custom logic for checking transaction limits in USD and performs an external call to the XVS token contract to burn tokens from the sender.
+- It overrides the `creditTo` function to trigger an external call to the XVS token contract to mint tokens for the receiver.
+- When sending tokens from this contract, it burns user tokens, with the burning logic residing in the XVS token contract.
+- When receiving tokens, it mints tokens for the receiver, with the minting logic residing in the XVS token contract.
```

Comment:
I know the logic under the hood and these words were not clear for me. Maybe we can add this:

```suggestion
- When sending tokens from the Destination Chain to the BNB chain, it burns user tokens, with the burning logic residing in the XVS token contract.
- When receiving tokens from the BNB chain (to the Destination Chain), it mints tokens for the receiver, with the minting logic residing in the XVS token contract.
```